Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/SpherePointer.cs

```diff
@@ -213,6 +227,15 @@ public bool TryGetNormalToNearestSurface(out Vector3 normal)
         /// </summary>
         private class SpherePointerQueryInfo
         {
+            // List of corners shared across all sphere pointer query instances --
+            // used to store list of corners for a bounds. Shared and static
+            // to avoid allocating memory each frame
+            private static List<Vector3> corners = new List<Vector3>();
+            // Help to clear caches when new frame runs
+            static private int lastRecalculation = -1;
+            // Map from grabbable => is the grabbable in FOV for this frame. Cleared every frame
+            private static Dictionary<Collider, bool> colliderCache = new Dictionary<Collider, bool>();
```

Comment:
Is this not going to be in shared location so touch can take advantage too?

Reply:
I plan to do this in a follow up PR. To make it shared I want to add some unit tests, so I was waiting to do that in follow up PR to keep this from getting too complex.